src/FlowtypeRunner.js

```diff
@@ -0,0 +1,48 @@
+const { exec } = require('child_process');
+const path = require('path');
+const { pass, fail } = require('create-jest-runner');
+
+class FlowtypeRunner {
+  constructor(globalConfig) {
+    this.globalConfig = globalConfig;
+  }
+
+  runTests(tests, watcher, onStart, onResult, onFailure, options) {
+    this.globalConfig = {};
+    const start = +new Date();
+    return new Promise((resolve) => {
+      exec('flow', { stdio: 'ignore', cwd: process.cwd() }, (err, stdout) => {
+        const errors = stdout.split('Error');
+        const errorsPerFile = errors.reduce((previous, current) => {
+          const firstErrorLine = current.split('\n')[0];
+          let fileName = firstErrorLine.split(':')[1];
+          if (fileName) {
```

Comment:
Do you use `[lints] all=warn` in your flowconfig? I *always* get yelled at for sketchy checks so I'm bringing it up here. What if you did something like (if fileName != null)?

Reply:
In the new code instead of file name I'm checking filenameMatcher (which is a `string.match(regex)` result. So any falsy value is false for me, thank you for explaining the idea for me. 😃